week2/AbstractExample/AbstractDemo.java

```diff
@@ -0,0 +1,27 @@
+public class AbstractDemo {
+    public static void main(String[] args) {
+        Student r1=new Student();
+        r1.setName("Anu");
+        r1.setAddress("NewDelhi");
+        System.out.println("Name is :"+r1.getName());

```

Comment:
Please use Abstraction as reference for all student, doctor and teacher

Reply:
fixed